src/control/logging/level.go

```diff
@@ -0,0 +1,94 @@
+//
+// (C) Copyright 2019 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+package logging
+
+import (
+	"fmt"
+	"strings"
+	"sync/atomic"
+)
+
+const (
+	// LogLevelDisabled disables any logging output
+	LogLevelDisabled LogLevel = iota
+	// LogLevelError emits messages at ERROR or higher
+	LogLevelError
+	// LogLevelInfo emits messages at INFO or higher
+	LogLevelInfo
+	// LogLevelDebug emits messages at DEBUG or higher
+	LogLevelDebug
+
+	disabledStr = "DISABLED"
```

Comment:
in the rest of the application I've used the equivalent of the convention strDisabled, do you want to add this which contradicts with the convention set out / change everything else to realign with this or adjust this to be in line with the rest of the application?